o4/o4.py

```diff
@@ -1135,6 +1139,16 @@ def main():
         args.insert(args.index('pyforce') + 1, '--')
     opts = docopt(__doc__, args)
 
+    if opts['version']:
+        from version import VERSION, VERSION_STR, TIMESTAMP, USER_EMAIL, USER_NAME
+        if not opts['--older']:
+            print(VERSION_STR)
+            print(TIMESTAMP.strftime('%Y-%m-%d'), USER_EMAIL, USER_NAME)
+            sys.exit(0)
+        testver = opts['--older'].split('.')
+        testver = tuple([int(d) for d in (testver + [0, 0, 0])[:3]])
```

Comment:
No `abs(int(sys.arg))` this time?

Reply:
If you say `--at-least one.three.seven`, you deserve the stack trace that you get.